esp/services/ws_sql/ws_sqlService.cpp

```diff
@@ -1468,6 +1468,7 @@ bool CwssqlEx::onPrepareSQL(IEspContext &context, IEspPrepareSQLRequest &req, IE
                 wu->setCloneable(true);
                 wu->setAction(WUActionCompile);
                 wu.setQueryText(ecltext.str());
```

Comment:
Good eye, but this notation is required due to the way the class is declared, it extends Owned<IWorkUnit>